doc/api/globals.md

```diff
@@ -637,6 +637,51 @@ logical processors available to the current Node.js instance.
 console.log(`This process is running on ${navigator.hardwareConcurrency} logical processors`);
 ```
 
+### `navigator.language`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string}
+
+The `navigator.language` read-only property returns a string representing the
+preferred language of the Node.js instance.
+
+The value is representing the language version as defined in [RFC 5646][].
+
+You can override the value of `navigator.language` by setting the
+`LC_ALL` environment variable.
+
+For example:
+
+```sh
+LC_ALL=fr-FR node --print "navigator.language" // 'fr-FR'
+```
+
+The fallback value on builds without ICU is `'en-US'`.
+
+```js
+console.log(`The preferred language of the Node.js instance has the tag '${navigator.language}'`);
+```
+
+### `navigator.languages`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Array<string>}
+
+The `navigator.languages` read-only property returns an array of strings
+representing the preferred languages of the Node.js instance.
+By default it contains only the value of `navigator.language`.
+The fallback value on builds without ICU is `['en-US']`.
+
+```js
+console.log(`The preferred languages are '${navigator.languages}'`);
```

Comment:
What is the default output? Can you add it as a comment like the other example?

If the user wanted to change the list of languages, how could they do so? Some value in `LC_ALL`? Can that be part of the example?

Reply:
done